### DISCUSSION TITLE
I can't sign up with GitHub in Vercel. It says "This user account is blocked."

### DISCUSSION THREAD

**Discussion by lianna947** [vercel/community#3288] [Help] [answered]
### Summary

I can't sign up with GitHub in Vercel. It says "This user account is blocked."

### Steps to Reproduce

.

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
